lib/savings.rb

```diff
@@ -0,0 +1,30 @@
+
```

Comment:
The file should probably be named `savings_account.rb` instead, but not a big deal.